kayak_core/src/context.rs

```diff
@@ -457,12 +642,33 @@ impl KayakContext {
         self.event_dispatcher.current_mouse_position()
     }
 
+    /// Query the Bevy `World` with the given `SystemParam`
+    ///
+    /// The function passed to this method will be called with the retrieved value from `World`. If
+    /// a value is returned from that function, it will be returned from this method as well.
+    ///
+    /// # Arguments
+    ///
+    /// * `f`: The function to call with the given system parameter
+    ///
+    /// # Examples
+    ///
+    /// ```ignore
+    /// use bevy::prelude::Res;
+    ///
+    /// struct MyCount(i32);
+    ///
+    ///#[widget]
```

Comment:
Can we show a second example here with multiple parameters?

Reply:
How's that?

Reply:
Looks good!